sdcm/nemesis.py

```diff
@@ -154,7 +155,7 @@ class CdcStreamsWasNotUpdated(Exception):
 
 class Nemesis:  # pylint: disable=too-many-instance-attributes,too-many-public-methods
 
-    disruptions_list: list[Callable] = []
+    disruptions_list: list[Callable]
```

Comment:
I guess this line can be removed completely

Reply:
removed